README.md

```diff
@@ -81,31 +85,37 @@ _Developers rarely spend the time to explain the pain point they face in detail
 
 Hi, I'm facing problem, please help.
 
-#### Steps to Reproduce 🕹
+#### Steps to reproduce 🕹
 
 1. Open https://codesandbox.io/s/recursing-mclean-2dub0?file=/demo.tsx
 2. Type <kbd>1</kbd> in the textbox. Once the options are loaded, the component displays options filtered by input value. The callback `onHighlightChange` log correct value `option 1 1`.
-3. Type <kbd>2</kbd> in the textbox. The component displays options filtered by input value. The callback `onHighlightChange` log wrong value `option 1 1` instead of `option 2 12`.
+3. Then type <kbd>2</kbd> in the textbox. The textbox now contains `12`. The component displays options filtered by input value. The callback `onHighlightChange` log wrong value `option 1 1` instead of `option 2 12`.
```

Comment:
```suggestion
3. Then type <kbd>2</kbd> in the textbox. The textbox now contains `12`. The component displays options filtered by input value. The callback `onHighlightChange` logs wrong value `option 1 1` instead of `option 2 12`.
```

Reply:
This typo was done on purpose, to simulate how non-native English speakers developer might write their issues 😁